build-system/common/exec.js

```diff
@@ -93,8 +93,10 @@ function execWithError(cmd) {
  */
 function execOrThrow(cmd) {
   const p = exec(cmd, {'stdio': ['inherit', 'inherit', 'pipe']});
-  if (p.stderr.length > 0) {
-    const error = new Error(p.stderr.toString());
+  if (p.status && p.status != 0) {
+    const error = new Error(
+      `Error executing \`${cmd}\: \n${p.stderr.toString()}`
+    );
```

Comment:
Can this function be consolidated with its namesake in `build-system/tasks/cherry-pick.js`?

https://github.com/ampproject/amphtml/blob/8c4c038e15e358345eb6f8548628896e68ebe4d0/build-system/tasks/cherry-pick.js#L23-L38

Reply:
@rsimha See question below re: shape of the function. My inclination is that `execOrThrow` should just take a command and throw the `stderr` if present, then calling code should catch or handle accordingly, but I could see the benefit of providing the error message directly as well. WDYT? I don't feel strongly in either direction.

Reply:
Good question. Right now, a majority of the existing use cases for `execOrThrow` are passing in an error message, which allows the caller to be more specific about what went wrong. I think we should retain this benefit.

I'm good with simply replacing the `execOrThrow` in `exec.js` with the version in `cherry-pick.js`.

Reply:
Done, PTAL